settings.js

```diff
@@ -131,6 +132,13 @@ function _migrateToV1(migrationSettings, extensionSettings) {
     let index = 0;
     const addedItems = new Set();
 
+    // Add the clubhouse icon
+    pages[Math.floor(index / itemsPerPage)][CLUBHOUSE_ID] = {
+        position: GLib.Variant.new_int32(index % itemsPerPage),
+    };
+    addedItems.add(CLUBHOUSE_ID);
+    index++;
```

Comment:
I think we're copy-pasting this code enough times now to be worth putting it in a separate function.